frontend/package.json

```diff
@@ -24,12 +24,14 @@
     "react-hook-form": "7.33.1",
     "react-redux": "8.0.2",
     "react-router-dom": "6.3.0",
-    "react-scripts": "5.0.1"
+    "react-scripts": "5.0.1",
+    "react-table": "7.8.0"
   },
   "devDependencies": {
     "@types/node": "18.0.5",
     "@types/react": "18.0.15",
     "@types/react-dom": "18.0.6",
+    "@types/react-table": "7.7.12",
```

Comment:
Have you reinstalled this dependencies using `--save-exact`, or just removed in `package.json` carets related to latest minor version involved on dependency? Looking into `package-lock` - it seems like you've not reinstalled them

Reply:
Do you mean (after removing carets) to "npm install --save-exact"?

Reply:
I mean you should run `npm uninstall <installed packages without fixed versions>` and then `npm i --save-exact <package>` and `npm i --save-exact --save-dev <dev_package>`